site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -99,7 +99,6 @@ export interface WorkspaceScheduleFormValues {
   ttl: number
 }
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
Removed it and ESLint was still happy 🤷🏼‍♂️ maybe it couldn't infer the type and now it can